ui/components/app/edit-gas-display/edit-gas-display.component.js

```diff
@@ -107,6 +108,12 @@ export default function EditGasDisplay({
       dappSuggestedAndTxParamGasFeesAreTheSame,
   );
 
+  let showTopWarning, warningMessage;
+  if (gasLimit < properGasLimit) {
+    showTopWarning = true;
```

Comment:
I don't think we need a `showTopWarning` variable -- on line 156, we can simply check if `warningMessage` is defined and display based on that.

Reply:
@darkwing Yes, you're right, we don't need a `showTopWarning` variable.
You can see changes here: b32a0fca871461945f501d2e3567d9d2cbb85c5e
Is it okay now?